client/service/README.md

```diff
@@ -0,0 +1,32 @@
+# Systemd files
+This folder contains scripts needed to create our own systemd unit service for DASHboard.
```

Comment:
This file needs to mention that `yarn run build` must be run each time the website is updated. This ran fine on my Pi 4 at home, but in the case that the Pi 3's can't run the build (ram issues, too slow, etc.) it may be simpler to suggest building it on a computer and copying it onto the Pi.

Reply:
good idea